integration-tests/src/test/java/tachyon/worker/DataServerIntegrationTest.java

```diff
@@ -107,6 +113,9 @@ private void assertValid(final DataServerMessage msg, final int expectedSize,
   @Before
   public final void before() throws IOException {
     System.setProperty(Constants.WORKER_DATA_SERVER, mDataServerClass);
+    if (!mNettyTransferType.equals(UNUSED_TRANSFER_TYPE)) {
+      System.setProperty(Constants.WORKER_NETTY_FILE_TRANSFER_TYPE, mNettyTransferType);
```

Comment:
Thanks for catching that! I added the clear in after(). I also got rid of the 'if' since it looks nicer and shouldn't change any behavior.